File: CS/WpfApplication/MyGridControl/MyGridControl.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.Xpf.Grid;

namespace WpfApplication.MyGridControl
{
    public class MyGridControl : GridControl
    {

        public MyGridControl()
        {

        }

        protected override DataViewBase CreateDefaultView()
        {
            return new MyCustomView();
        }
    }
}

File: VB/WpfApplication/MyGridControl/MyGridControl.vb
Imports DevExpress.Xpf.Grid

Namespace WpfApplication.MyGridControl

    Public Class MyGridControl
        Inherits GridControl

        Public Sub New()
        End Sub

        Protected Overrides Function CreateDefaultView() As DataViewBase
            Return New MyCustomView()
        End Function
    End Class
End Namespace
